src/js/views/patients/worklist/worklist_views.js

```diff
@@ -222,6 +225,62 @@ const EmptyFindInListView = View.extend({
   `,
 });
 
+const ListCountTemplate = hbs`
+  <strong>
+    {{#if isFlowList}}
+      {{formatMessage (intlGet "patients.worklist.worklistViews.countView.flowsCount") itemCount=count}}
+    {{else}}
+      {{formatMessage (intlGet "patients.worklist.worklistViews.countView.actionsCount") itemCount=count}}
+    {{/if}}
+  </strong>
+`;
+
+const MaximumCountTemplate = hbs`
+  <div>{{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumListCount") maximumCount=maximumCount listTypeLabel=listTypeLabel}}</div>
```

Comment:
these don't need the wrapping `<div>`s

Reply:
The `<div>`s are what make the two lines stack.

Without `<div>`s:

<img width="274" alt="Screenshot 2023-04-11 at 9 49 36 AM" src="https://user-images.githubusercontent.com/35355575/231201209-f34b5256-67d7-43a9-97ea-fe3d4a4614a1.png">

Or just remove the `<div>` around the `{{ formatMessage ... }}` portion?